File: back_lotteondisplay/src/main/java/com/plateer/lotteonDisplay/dto/cart/CartDetailDto.java
package com.plateer.lotteonDisplay.dto.cart;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.plateer.lotteonDisplay.dto.product.ImgJsn;
import com.plateer.lotteonDisplay.dto.product.ItmByMaxPurPsbJsn;
import com.plateer.lotteonDisplay.dto.product.ItmByMinPurJsn;
import lombok.*;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class CartDetailDto {
    private String cartSn;             //장바구니 번호

    private String spdNo;           //상품번호

    private String spdNm;           //상품이름

    private String sitmNo;          //단품번호

    private String sitmNm;          //단일상품 이름

    private String trGrpCd;         //거래처그룹코드

    private String trNo;            //거래처번호

    private String lrtrNo;          //하위거래처번호

    private String brdNm;           //브랜드 이름

    private List<ImgJsn> imgJsn;    //이미지

    private ItmByMaxPurPsbJsn itmByMaxPurPsbJsn;    //최대 수량

    private ItmByMinPurJsn itmByMinPurJsn;  //최소 수량

    private int slPrc;              //판매가

    private String estmtDlvTxt;     //도착문구

    private String mbNo;            //회원번호

    private int odQty;              //수량

    private String returnCode;      //응답 코드
}

File: front_lotteondisplay/src/model/dto/CartDetailDto.js
class CartDetailDto{
    constructor(cartSn, spdNo, spdNm, sitmNo, sitmNm, 
        trGrpCd, trNo, lrtrNo, brdNm, imgJsn, itmByMaxPurPsbJsn, itmByMinPurJsn,
        slPrc, estmtDlvTxt, mbNo, odQty){
        
        this.cartSn = cartSn;                   //장바구니 번호
        this.spdNo = spdNo;                     //상품번호
        this.spdNm = spdNm;                     //상품이름
        this.sitmNo = sitmNo                    //단품번호
        this.sitmNm = sitmNm;                   //단일상품 이름


        this.trGrpCd = trGrpCd;                 //거래처그룹코드
        this.trNo = trNo                        //거래처번호
        this.lrtrNo = lrtrNo;                   //하위거래처번호
        
        this.brdNm = brdNm;                     //브랜드 이름
        this.imgJsn = imgJsn;                   //이미지

        this.itmByMaxPurPsbJsn = itmByMaxPurPsbJsn; //최대 수량
        this.itmByMinPurJsn = itmByMinPurJsn;       //최소 수량


        this.slPrc = slPrc;                     //판매가
        this.estmtDlvTxt = estmtDlvTxt;         //도착문구

        this.mbNo = mbNo;                       //회원번호
        this.odQty = odQty;                     //수량
    }
}

export default CartDetailDto